content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -20,6 +20,7 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
   - [Changing Default Huge Page Size to 2 GiB](#changing-default-huge-page-size-to-2-gib)
   - [Support for ARM64 Platform](#support-for-arm64-platform)
   - [Replacing `backendStoreDriver` with `dataEngine`](#replacing-backendstoredriver-with-dataengine)
+  - [Recommending Updating Longhorn Nodes to Linux Kernel 5.19 or Higher](#recommending-updating-longhorn-nodes-to-linux-kernel-519-or-higher)
```

Comment:
The anchor link is not right.

should be `#updating-the-linux-kernel-on-longhorn-nodes`

Reply:
Updated. Thank you.